person.rb

```diff
@@ -1,3 +1,3 @@
 class Person
-
+  attr_accessor :id, :full_name, :phone_number, :city
```

Comment:
Bu sinif icin `constructor` methodu kullanman object-oriented programlama icin daha uygun. 

Ornek: http://stackoverflow.com/questions/8125946/class-constructors-in-ruby